setup.py

```diff
@@ -203,6 +203,7 @@
             'inspire_cache = inspirehep.modules.cache.ext:INSPIRECache',
             'inspire_search = inspirehep.modules.search:INSPIRESearch',
             'inspire_workflows = inspirehep.modules.workflows:INSPIREWorkflows',
+            'invenio_collections = invenio_collections:InvenioCollections',
```

Comment:
Also @jacquerie this PR uses the inspire_schemas modification with the schema and control_number field. So it will not pass anyway even if i comment out the line. I mean the test will fail not only because of the commented out line.